src/middleware/nasc.ts

```diff
@@ -95,6 +95,13 @@ async function NASCMiddleware(request: express.Request, response: express.Respon
 		return;
 	}
 
+	const nexAccount: HydratedNEXAccountDocument | null = await NEXAccount.findOne({ pid });
```

Comment:
Actually, this may cause issues with the path for account registration, since we also have a `nexAccount` there. Any suggestions?

Reply:
Yes this will break registration, as it will return an error if no NEX account is found. When a 3DS connects to our servers for the first time it doesn't have a PID yet and instead sends the password it would like to use for it's NEX account. Having this here will prevent any new consoles from registering

Honestly the best solution for now might be to just stop checking if the PID is linked to the device and that's it

Reply:
I can do that, once we fix the missing device edge case